src/planeta/StartHandshake.js

```diff
@@ -37,44 +79,15 @@ export default class Handshake extends React.Component {
     const { receipt } = this.state;
     const country = passport.country.fullName;
     const name = passport.data.name;
-    const url = "/planeta/handshake/" + receipt;
 
     return (
       <div>
```

Comment:
Can you refactor these divs (and some mor below on line 92 and onward) with rimble?

Reply:
What component would you suggest to use?

Reply:
`Flex` and `Text` :)